Stackage/Database.hs

```diff
@@ -666,6 +668,37 @@ getSnapshots l o = run $ (,)
         []
         [LimitTo l, OffsetBy o, Desc SnapshotCreated]
 
+getLtsSnapshots :: GetStackageDatabase m
+                => Int -- ^ limit
+                -> Int -- ^ offset
+                -> m (Int, [Entity Snapshot])
+getLtsSnapshots l o = run $ do
+    ltsCount <- count ([] :: [Filter Lts])
+    snapshots <- E.select $ E.from $
```

Comment:
@snoyberg BTW, does that query looks good? I am not an SQL expert, perhaps there is a better way.